task3/src/test/java/com/epam/vsharstuk/ObjectPoolTest.java

```diff
@@ -0,0 +1,26 @@
+package com.epam.vsharstuk;
+
+import org.testng.annotations.Test;
+
+
+public class ObjectPoolTest {
+
+    @Test
+    public void test() throws InterruptedException {
+        ObjectPool objectPool = new ObjectPool(10);
+
+        Thread consumerThread1 = new Consumer(objectPool, "-----Consumer1");
+        Thread producerThread1 = new Producer(objectPool, "-----Producer1");
+        //Thread consumerThread2 = new Consumer(objectPool, "=======Consumer2");
+        //Thread producerThread2 = new Producer(objectPool, "=======Producer2");
+
+        consumerThread1.start();
+        producerThread1.start();
+        //consumerThread2.start();
+        //producerThread2.start();
+        producerThread1.join();
+        consumerThread1.join();
+
```

Comment:
Good approach is to write tests like:
1. Create threads
2. Start them
3. Let them to work some time
4. Assert some collected by threads result

In current implementation we can only see logs